TheAdameia/Ada-s-Warehouse
# Front end structure

**Purpose**

Flesh out the form and functionality of the front-end application for all but the most speculative parts of MVP.

**Files Changed**

FloorController.cs, ItemController.cs - expanded controllers to accommodate front end logic
Floors.cs, Item.cs, ItemCategory.cs, UserProfile.cs - expanded models to accommodate front end logic
launchSettings.json (oversight corrected)
app.jsx, login, register, auth (setup)
managers - added following controller completion
Floors, Items - created display pages and front end logic
NavBar.jsx, ApplicataionViews.jsx - added support for pages

**To test**

Run the dotnet server. Make sure the database is up to date with the model changes. NPM run dev in the client folder.
